src/vs/workbench/parts/debug/electron-browser/debugService.ts

```diff
@@ -439,6 +442,19 @@ export class DebugService implements debug.IDebugService {
 		});
 	}
 
+	private debouncedDisplayThreads(session: RawDebugSession) {
+		let scheduler = this.displayThreadsScheduler.get(session.getId());
+		if (!scheduler) {
+			scheduler = new RunOnceScheduler(() => {
+				this.fetchThreads(session).done(undefined, errors.onUnexpectedError);
+			}, 100);
+			this.displayThreadsScheduler.set(session.getId(), scheduler);
```

Comment:
scheduler should be added to `toDisposeOnSessionEnd` so it gets properly disposed